spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/EntityGraphRepositoryMethodsIntegrationTests.java

```diff
@@ -96,22 +97,22 @@ void setup() {
 	@Test // DATAJPA-612
 	void shouldRespectConfiguredJpaEntityGraph() {
 
-		Assume.assumeTrue(currentEntityManagerIsAJpa21EntityManager(em));
+        assumeThat(currentEntityManagerIsAJpa21EntityManager(em)).isTrue();
```

Comment:
Some lines like this one require reformatting because the tabs indents were replaced with space indents.

Reply:
sorry for that, fixed in newest update